callstack/react-native-builder-bob
# feat(bob): add ignore pattern to config

### Summary

This adds the possibility to pass custom ignore patterns while working with builder bob.

### Test plan

1. Create a new library using `create-react-native-library`. You can select `Native module`, and `Kotlin & Swift` options.
2. Go to `package.json`, and find the `react-native-builder-bob` section
3. Replace `commonjs` target as following:
```json
{
    "exclude": "**/ignore_me/**"
    "targets": [
      "commonjs",
      "module",
      [
        "typescript",
        {
          "project": "tsconfig.build.json"
        }
      ]
    ]
}
```
4. Create a new folder named `src/ignore_me` and put `.ts` files inside it
5. Build the library and make sure the files inside the `ignore_me` aren't included in the `commonjs` and `module` targets.
